core/sql/sqlcomp/CmpDescribe.cpp

```diff
@@ -2355,10 +2355,134 @@ short CmpDescribeHiveTable (
       NAString colString(buf);
       Int32 j = i;
       outputColumnLine(space, colString, j);
-    }
+  }
 
   outputShortLine(space, "  )");
 
+  // show hive table partitions and buckets defination.
+  // this default schema name is what the Hive default schema is called in SeaHive
+  HiveMetaData* md = bindWA.getSchemaDB()->getNATableDB()->getHiveMetaDB();
+  NAString defSchema = ActiveSchemaDB()->getDefaults().getValue(HIVE_DEFAULT_SCHEMA);
+  defSchema.toUpper();
+  struct hive_tbl_desc* htbl;
```

Comment:
This variable doesn't seem to be used anywhere. Can we delete it?

Reply:
Deleted